resource_customizations/embed.go

```diff
@@ -6,5 +6,5 @@ import (
 
 // Embedded contains embedded resource customization
 //
-//go:embed *
+//go:embed all:*
```

Comment:
This allows embedding files that start with `_`, which doesn't happen by default.

Reply:
Should it be `all:*.lua` ?

Reply:
I tried a few different patterns, and they all said "no matching files found" 🤷